Makefile

```diff
@@ -191,6 +191,9 @@ list_test_data_options:
 lint:
 	pre-commit run --all-files
 
+gt4py_tests_gpu:
+	CUDA=y make build && \
+        docker run --gpus all $(FV3_IMAGE) python3 -m pytest -x gt4py
```

Comment:
Could this be generalized to both GPUs and CPUs?

Reply:
it could, but is there a need to run only the cpu ones when we can launch a gpu instance?

Reply:
meaning, before we use a new gt4py:develop, we probably want to require that both cpu and gpu tests work, and this runs both of them